code-challenges.test.js

```diff
@@ -33,21 +50,86 @@ const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
 
 // b) Create the function that makes the test pass.
+function divByThree(object) {
+  if (object.number % 3 === 0) {
+    return `${object.number} is divisible by three`
+  } else {
+    return `${object.number} is not divisible by three`
+  }
+}
+// PASS  ./code-challenges.test.js
+// divByThree
+//   ✓ should return '15 is divisible by three' when given { number: 15 } (1 ms)
+//   ✓ should return '0 is divisible by three' when given { number: 0 }
+//   ✓ should return '-7 is not divisible by three' when given { number: -7 }
+
+// Test Suites: 1 passed, 1 total
+// Tests:       3 passed, 3 total
+// Snapshots:   0 total
+// Time:        0.34 s, estimated 1 s
 
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
+// Pseudocode:
+//input:1 Array
+//output: 1 Array
+// Create a function letterCap that takes an array in and using the .map method to iterate through the array. Using the .charAt and the .to upperCase the first letter of each value and using .slice to replicate the rest of the  string and attach it to the uppercased first letter.
+
 // a) Create a test with expect statements for each of the variables provided.
 
+describe(letterCap, () => {
+  it("returns an array with the first letter in all the words capitalized", () => {
+    expect(letterCap(array)).toEqual([
```

Comment:
This is where you would want to plug in randomNouns1, and not use array as a param.